leshan-server-demo/webapp2/src/components/ClientSetting.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="pa-0">
+    <div title="Timeout used to send request">
+      <v-select
+        single-line
+        dense
+        hide-details
+        prefix="Timeout :"
+        :items="timeoutList"
+        item-value="val"
+        item-text="txt"
+        v-model="timeout"
+        append-outer-icon="mdi-help-circle-outline"
+        @click:append-outer="openTimeoutHelps"
+      ></v-select>
+    </div>
+    <div title="Content Format used to send request handling single value">
+      <v-select
+        single-line
+        dense
+        prefix="Format :"
+        hide-details
+        :items="singleFormatList"
+        v-model="singleformat"
+      ></v-select>
+    </div>
+    <!--v-select
+      single-line
+      dense
+      hide-details
+      prefix="Multi :"
+      :items="multiFormatList"
+      v-model="multiformat"
+    ></v-select-->
+  </div>
+</template>
+<script>
+import { preference } from "vue-preferences";
+
+export default {
+  data() {
+    return {
+      timeoutList: [
+        { txt: "5s", val: 5 },
+        { txt: "5min", val: 5 * 60 },
+        { txt: "15min", val: 15 * 60 },
+        { txt: "30min", val: 30 * 60 },
+        { txt: "None", val: null },
+      ],
+      singleFormatList: [
+        "TLV",
+        "CBOR",
+        "TEXT",
+        "OPAQUE",
+        "SENML_JSON",
+        "SENML_CBOR",
+        "JSON",
+      ],
+      //multiFormatList: ["TLV", "SENML_JSON", "SENML_CBOR", "JSON"],
+    };
+  },
+  computed: {
+    timeout: preference("timeout", { defaultValue: 5 }),
+    singleformat: preference("singleformat", { defaultValue: "TLV" }),
+    //multiformat: preference("multiformat", { defaultValue: "TLV" }),
```

Comment:
Commented code

Reply:
I comment  multi-node format content support as for now there is noway to write multi node with the new UI.
It should be activated later (I will keep it and add a comment)